styles.css

```
@@ -0,0 +1,75 @@
+*{
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+    background-color: #f4f4f4;
+}
+
+.parent {
+    display: grid;
+    grid-template: auto 1fr auto / auto 1fr auto;
+}
+
+header {
+    display: flex;
+    padding: 20px;
+    border-bottom: 2px solid #333; 
+    padding-bottom: 20px;
+    align-items: center;
+    justify-content: flex-start; 
+    padding-left: 20px;
+    padding-right: 20px;
+}
+
+header img {
+    margin-right: 15px; 
+}
+
+.left {
+    background: lightgray;
+    padding-left: 40px;
+    padding-right: 40px;
+    padding-top: 10px;
+    padding-bottom: 10px;
```

```suggestion
    [padding-left: 40px;
    padding-right: 40px;
    padding-top: 10px;
    padding-bottom: 10px;](padding: 10px 40px;)
```

- Cuando usa cuatro valores (padding: top right bottom left;), define cada lado individualmente.
- Cuando usa dos valores (padding: vertical horizontal;), el primer valor aplica a arriba y abajo, y el segundo a izquierda y derecha.